query/config.go

```diff
@@ -197,14 +198,22 @@ func (t *ConfigSummaryRequest) Parse() error {
 	return nil
 }
 
-func (t *ConfigSummaryRequest) configDeleteClause() string {
+func (t ConfigSummaryRequest) configDeleteClause() string {
 	if !t.Deleted {
 		return "config_items.deleted_at IS NULL"
 	}
 
 	return ""
 }
 
+func (t ConfigSummaryRequest) statusClause(q *gorm.DB) *gorm.DB {
+	if t.Status != "" {
```

Comment:
Can we use https://github.com/flanksource/duty/blob/main/query/commons.go#L13 ? 

I think we also need a health filter

That function can probably also be optimized to return equals / not equals instead of in / not in for single values